tests/tests/constants/compiledContracts/FINITE_LOOP_CONTRACT.json

```diff
@@ -0,0 +1,2127 @@
+{
+  "byteCode": "0x608060405234801561001057600080fd5b50600080819055506101f7806100276000396000f3fe608060405234801561001057600080fd5b50600436106100365760003560e01c806306661abd1461003b57806321b13c4814610059575b600080fd5b610043610075565b6040516100509190610100565b60405180910390f35b610073600480360381019061006e91906100c8565b61007b565b005b60005481565b60005b818110156100af576001600054610095919061011b565b6000819055506001816100a8919061011b565b905061007e565b5050565b6000813590506100c2816101aa565b92915050565b6000602082840312156100da57600080fd5b60006100e8848285016100b3565b91505092915050565b6100fa81610171565b82525050565b600060208201905061011560008301846100f1565b92915050565b600061012682610171565b915061013183610171565b9250827fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff038211156101665761016561017b565b5b828201905092915050565b6000819050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b6101b381610171565b81146101be57600080fd5b5056fea264697066735822122047c46dc26008bb379e9bf84c36d6616dd582a48a84efdc6de140b934805f7dfc64736f6c63430008030033",
```

Comment:
I'd remove the upper case for the file names.
I know it makes it feel like those are constants but I think it is too disruptive

Reply:
sounds good. So good ol camelCase?

Reply:
not for file names, the repo is using `name-like-this.ext` for non source files

Reply:
so as of now the script is sing the same name for the file and the contract so i means the contract will have lowercase as well (or I could add a toLowercase)

Reply:
weirdly, Im getting an error when using `-` 
<img width="550" alt="Screenshot 2021-04-15 at 15 18 22" src="https://user-images.githubusercontent.com/16758383/114894097-9ca93e00-9e0e-11eb-8aa2-8bdc1952fea0.png">

Reply:
"Test-Contract" (`-` means minus)

Reply:
It seems your wrote the file name without the `-`, Ex: finiteloopcontract.json is that a mistake or on purpose ?

Reply:
You should use `finite-loop-contract.json` and in yor ts:
```
{
  "finite-loop-contract" : ...
}
```

Reply:
The problem is that I'm using the same name for the contract name and `-` are not supported by SOlidity. I feel like using ThisFormat is pretty standard for Smart COntracts and their compiled json

Reply:
is `_` supported ?

Reply:
yes